### PR TITLE
Use High dpi pixmaps

### PR DESCRIPTION
**Pull request by SylvainCorlay**
This makes Spyder (even more) beautiful, especially with the new icon theme.

### CODE REVIEW

**Comment by SylvainCorlay** [comment]
With PyQt4, this has no effect.

File: spyderlib/spyder.py [hunk 1]
```diff
@@ -272,6 +272,7 @@ def __init__(self, options=None):
         QMainWindow.__init__(self)
 
         qapp = QApplication.instance()
+        qapp.setAttribute(Qt.AA_UseHighDpiPixmaps)
```
Comment (ccordoba12) [inline spyderlib/spyder.py]: Does PyQt4 have a property called `AA_UseHighDpiPixmaps`?

Comment (SylvainCorlay) [inline spyderlib/spyder.py]: Actually it does not. I was mistaken.

File: spyderlib/spyder.py [hunk 2]
```diff
@@ -272,6 +272,8 @@ def __init__(self, options=None):
         QMainWindow.__init__(self)
 
         qapp = QApplication.instance()
+        if PYQT5:
+            qapp.setAttribute(Qt.AA_UseHighDpiPixmaps)
         self.default_style = str(qapp.style().objectName())
```
Comment (goanpeca) [inline spyderlib/spyder.py]: Yes much better!

**Comment by goanpeca** [comment]
:+1:

File: spyderlib/spyder.py (hunk 2 above)
Comment (ccordoba12) [inline spyderlib/spyder.py]: Yep, this is nicer!

**Pull request merged**